addon-resizer/nanny/main/pod_nanny.go

```diff
@@ -60,6 +60,7 @@ var (
 	pollPeriod     = time.Millisecond * time.Duration(*flag.Int("poll-period", 10000, "The time, in milliseconds, to poll the dependent container."))
 	estimator      = flag.String("estimator", "linear", "The estimator to use. Currently supported: linear, exponential")
 	minClusterSize = flag.Uint64("minClusterSize", 16, "The smallest number of nodes resources will be scaled to. Must be > 1. This flag is used only when an exponential estimator is used.")
+	useMetrics     = flag.Bool("use-metrics", false, "Whether to use Kubernetes API or apiserver metrics to detect cluster size")
```

Comment:
Leaving the @bskiba whether this should be documented in README.md

Reply:
It would be nice to drop a line, yes.

Reply:
done